WordPress/src/main/java/org/wordpress/android/models/ReaderPost.java

```diff
@@ -39,8 +40,9 @@
     private String primaryTag;    // most popular tag on this post based on usage in blog
     private String secondaryTag;  // second most popular tag on this post based on usage in blog
 
-    public double sortIndex;
-    private String published;
+    public double sortIndex;      // determines how posts are sorted for display
+    private String date;          // generic date - depends on the stream the post is from
```

Comment:
Can you be more specific in this comment? Add an example use case (like `which is set to the tagged date when available`)?